packages/miniflare/src/workers/assets/assets.worker.ts

```diff
@@ -1,5 +1,4 @@
-// @ts-ignore the fact that there are no typings for this module
-import worker from "@cloudflare/workers-shared/dist/asset-worker.mjs";
+import worker from "@cloudflare/workers-shared/asset-worker/src/index";
```

Comment:
how come we're not importing the built Workers anymore?

Reply:
miniflare will build anyway